src/commands/key/sign.ts

```diff
@@ -0,0 +1,226 @@
+import { Command, Flags } from "@oclif/core";
+import { getWalletForAddress } from "../../wallet";
+import { NetworkName, NetworkNames, Networks } from "../../networks";
+import { loadAbi, ContractName } from "../../contracts";
+import { loadWallet } from "../../keystore";
+import { ethers } from "ethers";
+import { TransactionCommand } from "../../base";
+import { getSigner } from "../../helpers";
+import { TypedDataSigner } from "@ethersproject/abstract-signer";
+import { signerToEcdsaValidator } from "@zerodev/ecdsa-validator";
+import { createPublicClient, http } from "viem";
+import { sepolia } from "viem/chains";
+import { KERNEL_V3_1 } from "@zerodev/sdk/constants";
+import { entryPoint07Address } from "viem/account-abstraction";
+import { privateKeyToAccount } from "viem/accounts";
+
+export default class Sign extends TransactionCommand {
+  static description = "Generate an approval signature for deploySite";
+
+  static examples = [
+    `$ earthfast key sign --network testnet-sepolia --wallet 0x123... --password mypassword --amount 10 --deadline 1742434970 --token USDC`,
```

Comment:
not sure if you considered this but `getSigner` should have the wallet which contains the privateKey. this way the current paradigms of wallet selection from the list with password prompt and the `--key` CLI param would work

for example 
```
const signer = await getSigner(flags.network, flags.rpc, flags.address, flags.signer, flags.key, flags.account);
const privateKey = ((signer as Wallet).privateKey);
```

Reply:
also same in execute.ts

Reply:
Updated accordingly